tests/hazmat/primitives/test_dh.py

```diff
@@ -18,6 +19,11 @@
 from ...utils import load_nist_vectors, load_vectors_from_file
 
 
+skip_no_dhx = pytest.mark.skipif(
```

Comment:
A docker container with ubuntu 14.04 will work for that purpose. Hopefully that's a relatively easy path for you!